lib/editor/result.coffee

```diff
@@ -87,18 +96,19 @@ class Result
 
   # Bulk Actions
 
-  @forLines: (ed, start, end) ->
-    ed.findMarkers().filter((m)->m.result? &&
-                                 m.getBufferRange().intersectsRowRange(start, end))
-                    .map((m)->m.result)
+  @forLines: (ed, start, end, type = 'any') ->
+    ed.findMarkers().filter((m) -> m.result? &&
+                                   m.getBufferRange().intersectsRowRange(start, end) &&
+                                  (m.result.type == type || type == 'any'))
+                    .map((m) -> m.result)
```

Comment:
Probably, but it seems a tad more convenient to me. But we can do it either way, I don't care too much about it at all ;)